guides/puppet-web-apps/html-template-guidelines/README.md

```diff
@@ -0,0 +1,157 @@
+
+# HTML Template Guidelines
+
+In order to harness the full power of Starcounter, applications should be built to accomodate for complete visual and functional interoperability. To make this process easier for developers, we provide these guidelines for HTML templates which, when followed, will allow applications to achieve seamless visual integration with other applications.

```

Comment:
What do you think about using the term "HTML views", not "HTML templates"? See: https://github.com/Starcounter/TechComm/issues/286